exercises/java/src/main/java/cards/FaceValue.java

```diff
@@ -1,39 +1,14 @@
 package cards;
 
-public enum FaceValue {
```

Comment:
Looks good but I reckon if we're going this far then we might as well ditch the whole FaceValue class and just have it as a String attribute on Card?

Reply:
🤷‍♂️  I’d be up for that. Was there an intention of introducing them to enums?

Reply:
Nope let's just bin the whole class

Reply:
Done in a4f7c29e8587e9b04bd5f009bffd53993f52c00a